src/script/components/list/GroupedConversationHeader.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Wire
+ * Copyright (C) 2019 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import React from 'react';
+import cx from 'classnames';
+
+import {registerReactComponent} from 'Util/ComponentUtil';
+
+import type {ConversationLabel} from '../../conversation/ConversationLabelRepository';
+import NamedIcon from '../NamedIcon';
+
+export interface GroupedConversationHeaderProps {
+  conversationLabel: ConversationLabel;
+  isOpen: boolean;
+}
+
+const GroupedConversationHeader: React.FC<GroupedConversationHeaderProps> = ({conversationLabel, isOpen}) => {
+  const badge = conversationLabel.conversations().filter(conversation => conversation.hasUnread()).length;
+
+  return (
+    <div
+      className={cx('conversation-folder__head', {'conversation-folder__head--open': isOpen})}
+      data-uie-name="conversation-folder-head"
+    >
+      <div className="disclose-icon">
+        <NamedIcon name="disclose-icon" width="5" height="8" />
```

Comment:
```suggestion
        <NamedIcon name="disclose-icon" />
```
width="5" height="8" i'm not sure you need these for this component or not, you should check what the current rendered size is by inspecting on dev and pass those values...

Reply:
also not sure why the className of disclose-icon is passed to the upper div is it intended?
![image](https://user-images.githubusercontent.com/63250054/106921585-691caa00-6721-11eb-807f-de5232561558.png)

Reply:
I checked it and they are needed - that's why I added them ;)

Reply:
And yes the upper div is needed, otherwise some CSS styles like `.disclose-icon svg` won't apply.